Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:latest
+
+LABEL maintainer="haisam.ido@gmail.com"
+
+RUN mkdir openmct
+WORKDIR /openmct
+
+# Mounting a volume would be better
+ADD . .
```

Comment:
In nodejs projects it is recommended to copy package.json alone first, install node modules and only then the rest of the code to avoid rebuilding all of the layers every time you change some file.